
# topic is try and exception

# try:
#     print(x)
# except:
#     print("an exception occured")



# try:
#     if a > 10:
#         print("i am trying to hack you")

# except:
#     print("bro are you ok")

try:
    print(x)
except Exception as e:
    print(e)